docs/ref/models/relations.txt

```diff
@@ -135,12 +135,31 @@ Related objects reference
         :class:`~django.db.models.ForeignKey`\s where ``null=True`` and it also
         accepts the ``bulk`` keyword argument.
 
+    .. method:: set(obj1, [obj2, ...])
```

Comment:
Ok now I remember why I didn't add it for `bulk`, it's not available on all relational managers.